src/shared/services/map-location/index.js

```diff
@@ -1,28 +1,60 @@
-function mapLocation(loc) {
+export const location2feature = location => ({
+  type: 'Point',
+  coordinates: [location.lng, location.lat],
+});
+
+export const feature2location = feature => {
+  const { coordinates } = feature;
+  return {
+    lat: coordinates[1],
+    lng: coordinates[0],
+  };
+};
+
+export const address2pdok = address => {
+  const { openbare_ruimte, huisnummer, huisletter, huisnummer_toevoeging, postcode, woonplaats } = address;
+
+  return {
+    straatnaam: openbare_ruimte,
```

Comment:
Is this correct? Can a value for `openbare_ruimte` be considered the same as `straatnaam`?

Reply:
yes!